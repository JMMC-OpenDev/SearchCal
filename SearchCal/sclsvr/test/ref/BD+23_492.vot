version https://git-lfs.github.com/spec/v1
oid sha256:c398045c4e25365091e10bd3f10aaad769bcd0c05fbf5ad7e593c4cf9fb16edf
size 230788
