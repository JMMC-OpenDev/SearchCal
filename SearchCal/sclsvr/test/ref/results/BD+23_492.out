version https://git-lfs.github.com/spec/v1
oid sha256:e45dcd7d27a821995e882c3df01193b2f5c3637eaa08cff48a48cf9832980a0f
size 144848
